platform/android/java/lib/src/org/godotengine/godot/Godot.kt

```diff
@@ -336,12 +337,14 @@ class Godot(private val context: Context) : SensorEventListener {
 		}
 
 		if (nativeLayerInitializeCompleted && !nativeLayerSetupCompleted) {
-			nativeLayerSetupCompleted = GodotLib.setup(commandLine.toTypedArray(), tts)
+			var audioManager: AudioManager = context.getSystemService(Context.AUDIO_SERVICE) as AudioManager
+			nativeLayerSetupCompleted = GodotLib.setup(commandLine.toTypedArray(), tts, audioManager)
```

Comment:
`context.getSystemService(Context.AUDIO_SERVICE)` can be called on the C++ side, but the code would be harder to understand and longer, this is just here for readability. Will change if needed